src/libs/actions/IOU.ts

```diff
@@ -3685,6 +3686,14 @@ function cancelPayment(expenseReport: OnyxTypes.Report, chatReport: OnyxTypes.Re
         },
     ];
 
+    if (!isFree) {
```

Comment:
Hmm why no optimistic next step for free policies? Can you explain in a comment too so it's clear?

Reply:
We don't show nextSteps for free policies 😁

Reply:
didn't know that for newdot since it's different for olddot!! noice!